packages/core-design-system/scripts/constants.js

```diff
@@ -0,0 +1,45 @@
+import { formats } from 'style-dictionary/enums'
+
+export const DESIGN_SYSTEM_ROOT = 'src/styles'
+export const DESIGN_SYSTEM_ROOT_ESM = 'src/styles-esm'
+
+export const DESIGN_SYSTEM_PREFIX = 'cn'
+
+export const STYLE_BUILD_FORMATS = {
+  CSS_VARIABLES: formats.cssVariables,
+  JS_ESM: formats.javascriptEsm
```

Comment:
Since everything that's put here is `.ts` files, it's neither JS, nor ESM, so this is a bit confusing